site/page.js

```diff
@@ -26,10 +26,10 @@ class Page extends React.Component {
           />
           <link rel="canonical" href={prefixUrlAbsolute(this.props.path)} />
         </Helmet>
-        <div className="overflow-auto viewport-full-mm w180-mm fixed-mm top left flex-parent-mm flex-parent--stretch-cross-mm">
+        <div className="overflow-auto h-viewport-full-mm w180-mm fixed-mm top left flex-parent-mm flex-parent--stretch-cross-mm">
           <Navigation />
         </div>
-        <div className="ml180-mm limiter w-auto pl60-mm pr60-mm mb60 mt24">
+        <div className="ml180-mm w-full px24 w-11/12-mm wmax1200-mm px60-mm mb60 mt24">
```

Comment:
This is not working correctly. There's vertical scroll and a lot of the page is cut off. We could do something like this:

```suggestion
        <div className="ml180-mm px24 wmax1200-mm px60-mm mb60 mt24">
```

But then you're not getting a nice proportionally sized container.

Reply:
Whoops! Want to have another look? We _should_ be able to do this without introducing more stuff.